change/@fluentui-react-b5938547-532d-41fa-b34e-d8d702171623.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Updated onChange prop, added initial date, use new getDateFromTimeSelection function",
```

This goes in the release notes, so it's good to include the control name
```suggestion
  "comment": "TimePicker: Updated onChange prop, added initial date, use new getDateFromTimeSelection function",
```

---

Resolved by adding control name.